### DISCUSSION TITLE
2023/12/13/lua-express

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#33] [Blog comments]
# 2023/12/13/lua-express

lua-express-banner

https://blog.amd-nick.me/2023/12/13/lua-express

**Comment by BrownJourney** [comment]
Когда тебе настолько противен JS, что ты готов переписать фреймворк на любим Lua.

**Reply by AMD-NICK** [reply]
Ага. Это максимально глупо, но я видел путь и не видел препятствий
